tools/wptserve/wptserve/handlers.py

```diff
@@ -467,6 +467,8 @@ def __init__(self, path, format_args, content_type, **headers):
         :param headers: List of headers to send with responses"""
 
         with open(path) as f:
-            data = f.read() % format_args
```

Comment:
This change may not be wrong, but I'm not clear what it has to do with the rest of the PR?

Reply:
Without this change we can't setup the print-reftest harness page without passing in empty format args. Note it's a separate commit in the series